### DISCUSSION TITLE
Couldn't join repo

### DISCUSSION THREAD

**Discussion by andrearussomando** [community/community#22080] [New to GitHub]
<p>Dear all<br>
I received an invitation to join a private repo<br>
I created an account (with the same email that received the invitation) but I cannot join it<br>
If I press Accept invitation, I got error 404<br>
I tried to follow this suggestions, but were not helpful</p><aside class="quote" data-post="3" data-topic="1532">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/got-an-invitation-to-join-repo-but-cant-get-past-404/1532/3">Got an invitation to join repo, but can't get past 404</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    If the invitation wasn’t retracted like <a class="mention" href="/wabri">@wabri</a> mentioned, try: 

Log in
Navigate directly to the repository

If the invitation is still active, you will then be greeted by a “you’re invited” screen. 
Also make sure that the account you are logged in with is the same account which received the invite!
  </blockquote>
</aside>

<p>(I just received the invitation, so I don’t think it is retracted, I logged with the account that received the mail, I’m unable to navigate directly to the repo, it is private)</p>
<p>Any help will be appreciated</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>I think the next thing I’d try is accepting the invitation in a Private Browsing or Incognito window to make sure that a browser extension isn’t interfering. To do that, follow these steps:</p>
<ul>
<li>Open the Incognito or Private Browsing window and login to your GitHub account</li>
<li>Copy the entire link in your invitation email.</li>
<li>Paste it into the Incognito/Private browsing browser window were you are logged in</li>
</ul>
<p>If that doesn’t do it, then please contact support directly so they can see if there’s any account related reasons this may be failing.</p>
<p><a href="https://support.github.com/contact" class="onebox" target="_blank">https://support.github.com/contact</a></p>

**Comment by andrearussomando** [comment]
<p>Thanks for the help<br>
At the end the invitation was send again, and this solved the issue</p>
